src/Auth/Manager.php

```diff
@@ -171,9 +171,10 @@ public function hasUser()
      * Sets the user
      * @phpstan-param Models\User $user
      */
-    public function setUser(Authenticatable $user)
+    public function setUser(Authenticatable $user): self
```

Comment:
```suggestion
    public function setUser(Authenticatable $user): static
```

Reply:
@bennothommo Laravel 11 uses "self" in their implementation:
```php
    /**
     * Sets the user
     * @phpstan-param Models\User $user
     */
    public function setUser(Authenticatable $user): self
...
```

Reply:
We don't have to follow that unless it's part of the contract (AFAIK, it's not). The linked article says that `static` is "broader" than `self`, and would allow extensions of the `AuthManager` to narrow the scope.